packages/core/src/Animations.ts

```diff
@@ -61,6 +62,18 @@ export const snapPoint = (
 type Dependencies = readonly unknown[];
 type Atomic = string | number | boolean;
 
+export const useVector = (x: number, y: number, deps: Dependencies) =>
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useMemoOne(() => vec.createValue(x, y), [deps]);
+
+export const useClock = (deps: Dependencies) =>
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useMemoOne(() => new Clock(), [deps]);
+
+export const useValue = <V extends Atomic>(value: V, deps: Dependencies) =>
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useMemoOne(() => new Value(value), [deps]);
+
```

Comment:
Should not be pass `deps` directly to the `useMemoOne` hook instead of wrapping it into another array? Like in the `useValues` hook?

Reply:
Nice catch thank you 😳